libraries/AP_Networking/AP_Networking_Config.h

```diff
@@ -107,6 +107,10 @@
 #define AP_NETWORKING_CAN_MCAST_ENABLED 0
 #endif
 
+#ifndef AP_NETWORKING_CAN_MCAST_BRIDGING_ENABLED
+#define AP_NETWORKING_CAN_MCAST_BRIDGING_ENABLED AP_NETWORKING_CAN_MCAST_ENABLED && !defined(HAL_BOOTLOADER_BUILD)
```

Comment:
The default for the bootloader should be in `libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_bootloader.h`

Reply:
fixed